src/mode/modeHandler.ts

```diff
@@ -1211,7 +1211,7 @@ export class ModeHandler implements vscode.Disposable {
     if (vimState.currentMode === ModeName.Normal && !recordedState.operator) {
       for (const { stop, i } of Range.IterateRanges(vimState.allCursors)) {
         if (stop.character >= Position.getLineLength(stop.line)) {
-          vimState.allCursors[i].withNewStop(
+          vimState.allCursors[i] = vimState.allCursors[i].withNewStop(
```

Comment:
hahaha there's the cause of so many of our test errors. Is there not some kind of lint setting for this?